docs/real-time-analytics/discounting/manual.sql

```diff
@@ -0,0 +1,33 @@
+-- Discount Codes
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('ALDNE15', 15.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('LWNDB15', 15.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('QQNSH15', 15.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('ENDJW15', 15.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('MWJDS15', 15.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('WVESJ20', 20.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('VNNSW20', 20.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('XSNPT20', 20.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('JDNTY25', 25.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('KIDNT25', 25.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('PEHTN25', 25.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('BSLWU30', 30.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('URITN40', 40.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('LSEMJ50', 50.0);
+INSERT INTO discount_codes (CODE, PERCENTAGE) VALUES ('FKENT50', 50.0);
+
+-- Orders
+INSERT INTO orders (ID, ORDER_TIME, CUSTOMER_ID, ITEM_CT, ORDER_SUBTOTAL, DISCOUNT_CODE) VALUES (1000, '2021-12-03T14:12:01', 101,  3, 201.17, 'LSEMJ50');
```

Comment:
Instead of `'2021-12-03T14:12:01'` should we use `UNIX_TIMESTAMP()` -- if the recipe becomes older, it could become an issue to use old timestamps, as default retention time of 7 days would result in immediate deletion of the data in the input topic...